src/components/views/right_panel/TimelineCard.tsx

```diff
@@ -73,7 +74,7 @@ export default class TimelineCard extends React.Component<IProps, IState> {
     constructor(props: IProps) {
         super(props);
         this.state = {
-            showReadReceipts: false,
+            showReadReceipts: SettingsStore.getValue("showReadReceipts", props.room.roomId),
```

Comment:
I maybe should we add a listener for the setting?

Reply:
It's nice to use listeners for quick update to settings changes, but we're also quite inconsistent about it throughout, so whatever seems best to you is okay.